packages/core/src/fhir-to-cda/cda-templates/components/__tests__/xmls/allergy-section-single-entry.xml

```diff
@@ -0,0 +1,103 @@
+<templateId root="2.16.840.1.113883.10.20.22.2.6.1" extension="2015-08-01">
+</templateId>
+<code code="48765-2" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC" displayName="Allergies and Adverse Reactions">
+</code>
+<title>
+	ALLERGIES, ADVERSE REACTIONS, ALERTS
+</title>
+<text>
+	<table ID="allergies">
+		<thead>
+			<tr>
+				<th>
+					Substance
+				</th>
+				<th>
+					Category
+				</th>
+				<th>
+					Reaction
+				</th>
+				<th>
+					Date of Occurrence
+				</th>
+				<th>
+					Date of First Onset
+				</th>
+				<th>
+					Comments
+				</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr ID="allergies1">
+				<td>
+					Penicillin G
+				</td>
+				<td>
+					medication
+				</td>
+				<td ID="allergies1-reaction">
+					Hives
+				</td>
+				<td>
+					02/26/2008 8:05 AM
+				</td>
+				<td>
+					05/01/1998 12:00 AM
+				</td>
+				<td>
+					Not Specified
```

Comment:
should be "rash, itching/swelling and trouble breathing"? some sort of issue with parsing logic?

Looking farther its because your pushing an empty note [here](https://github.com/metriport/metriport/pull/2208/files#r1628390792)

Reply:
Ya, just checking that not including the note doesn't break the rest of the logic